.github/CODEOWNERS

```diff
@@ -1,41 +1,40 @@
 # Each line is a file pattern followed by one or more owners.
 
 # Current reviewers-XXX teams, who review everything for approval.
-#* @reviewers-amazon @reviewers-apple @reviewers-comcast @reviewers-google @reviewers-samsung
-* @chrisdecenzo @rwalker-apple @bzbarsky-apple @hawk248 @jelderton @robszewcyk @mspang @saurabhst @BroderickCarlin
+* @project-chip/reviewers-amazon @project-chip/reviewers-apple @project-chip/reviewers-comcast @project-chip/reviewers-google @project-chip/reviewers-samsung
```

Comment:
I want to say there was a reason we didn't use this approach. IIRC @woody-apple may have more background here

Reply:
So we prefer to list  @chrisdecenzo @rwalker-apple @bzbarsky-apple @hawk248 @jelderton @robszewcyk @mspang @saurabhst @BroderickCarlin rather than groups?

Reply:
Oh, I think I see why - GitHub tries to load balance and does have an "all" route.